doc/logos/README.md

```diff
@@ -0,0 +1,47 @@
+# scikit-learn Branding
+
+This section contains scikit-learn's branding standards.
+File types:
+- `SVG` Scalable Vector Graphics (SVG) are an XML-based markup language for describing two-dimensional based vector graphics. They can be created and edited with any text editor or with drawing software.
+- `PNG` is a higher-quality compression format; size of file is generally larger
+- `ICO` file format refers to an image file format that contains small size computer icon images
+
+---
+
+## scikit-learn: Font, Colors & Logos
+
+### scikit-learn Font
+- "scikit": Helvetica Neue
+- "learn": Script MT
+
+### scikit-learn Color Palette
+![#29ABE2 Cyan](brand_colors/colorswatch_29ABE2_cyan.png) `RGB 41/171/226 | HEX #29ABE2 | scikit-learn Cyan` | More info: [#29ABE2](https://www.color-hex.com/color/#29abe2)
+
+![#F7931E Orange](brand_colors/colorswatch_F7931E_orange.png)  `RGB 247/147/30 | HEX #F7931E | scikit-learn Orange` | More info: [#F7931E](https://www.color-hex.com/color/#f7931e)
+
+![#9B4600 Brown](brand_colors/colorswatch_9B4600_brown.png) `RGB 155/70/0| HEX #9B4600 | scikit-learn Brown` | More info: [#9B4600](https://www.color-hex.com/color/#9b4600)
+
+### scikit-learn Logos
+
+#### Logo 1
+- File type: PNG
+- File size: 49 KB (1280 x 689 px)
+- File name: [1280px-scikit-learn-logo.png](https://github.com/scikit-learn/scikit-learn/blob/main/doc/logos/1280px-scikit-learn-logo.png)
+
+<img src="1280px-scikit-learn-logo.png" width="200">
+
+#### Logo 2
+- File type: ICO
+- File size: 2 KB (32 x 32 px)
+- File name: [favicon.ico](https://github.com/scikit-learn/scikit-learn/blob/main/doc/logos/favicon.ico)
+
+<img src="favicon.ico" width="100">
+
+
```

Comment:
Can we include the SVG here too?

```html
<img src="scikit-learn-logo-without-subtitle.svg" width="250px">
```

The width can be anything, I picked 250px to have a similar size to the others.

Reply:
Done.